### ISSUE TITLE
Assert crashes if trying to update an empty physics world

### ISSUE THREAD

**Issue by haikarainen** [DanielChappuis/reactphysics3d#30]
A simple check at the top of the update function fixes this for me:

```c++
// Update the physics simulation
/**
 * @param timeStep The amount of time to step the simulation by (in seconds)
 */
void DynamicsWorld::update(decimal timeStep) {

    if (mBodies.size() < 1)
    {
      return;
    }

    // Rest of function goes here ...
```

**Comment by DanielChappuis** [comment]
Thanks for reporting this. This issue has already been cited here : https://github.com/DanielChappuis/reactphysics3d/issues/26

This will be fixed in the next release 0.7 of the library.

**Comment by DanielChappuis** [comment]
This issue has been fixed in the version v0.7.0 of ReactPhysics3D that has just been released.
Thanks a lot for reporting this.
